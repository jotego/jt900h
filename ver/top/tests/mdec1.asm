    ; Module decrement 1
    main section code
    org 0
    ld a,0xbf    ; common header

    ld wa,0x1f
    mdec1 4,wa
    ld b,a
    mdec1 4,wa
    ld c,a
    mdec1 4,wa
    ld d,a
    mdec1 4,wa
    ld e,a
    mdec1 4,wa
    ld h,a
test_end:
    ld (0xffff),0xff
data:
    dw 0xcafe,0xbeef,0xffff,0xeeee,0xcccc
    end