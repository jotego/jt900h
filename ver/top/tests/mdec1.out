0000001e - 00000000 - 00000000 - 00000000
00001e1d - 00000000 - 00000000 - 00000000
00001c1f - 00000000 - 00000000 - 00000000
00001e00 - 00000000 - 00000000 - 00000000
00000000 - 00000000 - 00000000 - 00000000
